### PR TITLE
Add iscsi-direct pool for find-storage-pool-sources/-as

### PR DESCRIPTION
**Pull request by meinaLi**
Add iscsi-pool for find-storage-pool-sources/-as
Signed-off-by: meinaLi <meili@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
Please paste test result

**Comment by meinaLi** [comment]
find-storage-pool-sources:
avocado run --vt-type libvirt virsh.find_storage_pool_sources..iscsi_direct_source..
JOB ID     : b95096fc5c75bcbd4f777577602daece70ac2cf5
JOB LOG    : /root/avocado/job-results/job-2019-07-17T03.01-b95096f/job.log
 (1/2) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.non_acl.remote_host.iscsi_direct_source.specified_srcSpec: PASS (41.21 s)
 (2/2) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.acl_test.remote_host.iscsi_direct_source.specified_srcSpec: PASS (42.32 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 86.28 s

find-storage-pool-sources-as:
avocado run --vt-type libvirt virsh.find_storage_pool_sources_as..iscsi_direct_type
JOB ID     : f643e465291dea0c02bbeb9decfdc19eb89c3d14
JOB LOG    : /root/avocado/job-results/job-2019-07-17T03.10-f643e46/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources_as.positive_test.local_source.iscsi_direct_type: PASS (37.57 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 40.21 s

File: libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def run(test, params, env):
 
     source_type = params.get("source_type", "")
     source_host = params.get("source_host", "127.0.0.1")
+    source_initiator = params.get("source_initiator", "iqn.2019-07.com.example:client")
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py]: I think default should be ""

Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py]: Do you need default value here? I notice there is no default value set  on line 28 in virsh_find_storage_pool_sources_as.py

Comment (meinaLi) [reply libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py]: Updated

Comment (meinaLi) [reply libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py]: Yes, no need to use the value and I have already modified it.

**Pull request merged**